src/shipit_pulse_listener/default.nix

```diff
@@ -24,6 +24,29 @@ EOF
     '';
   });
 
+  mkBot = branch:
+    let
+      secretsKey = "repo:github.com/mozilla-releng/services:branch:" + branch;
+    in
+      mkTaskclusterHook {
+        name = "Triggers Taskcluster hooks on pulse messages";
+        owner = "babadie@mozilla.com";
+        schedule = [ "0 0 * * * *" ];  # every hour
```

Comment:
Does this mean that we will process all Pulse messages only once every hour? I ask because we'd like to keep the time between a *review request* and its *static analysis feedback response* below 15min if possible.